### DISCUSSION TITLE
V script mode: Minor improvements

### DISCUSSION THREAD

**Discussion by IngwiePhoenix** [vlang/v#11804] [Ideas]
Basically:

- `v sh myscript.vsh` for readability to distinguish against `v run`
- `v sh --root=modname myscript.vsh` to allow someone to script in the namespace of another module. Might be useful for utility scripts.

Those are two things that I had wished V had, so I thought I'd just post them here. :)

**Comment by litenite42** [comment]
> v sh myscript.vsh for readability to distinguish against v run

you shouldn't need to directly invoke v here? 
place `#!/usr/local/bin/v run` at the top of the script and you should be able to do just `./myscript.vsh`

unless you're talking about the directive, then idk. i think run is still the best since a script is essentially just a single file v program with everything wrapped in `fn main() {...}` (iirc).

The second point definitely sounds interesting.
